scripts/govcms-deploy

```diff
@@ -10,30 +10,34 @@ LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
 GOVCMS_DEPLOY_WORKFLOW_CONFIG=${GOVCMS_DEPLOY_WORKFLOW_CONFIG:-import}
 # Determine the content strategy `import` vs `retain`.
 GOVCMS_DEPLOY_WORKFLOW_CONTENT=${GOVCMS_DEPLOY_WORKFLOW_CONTENT:-retain}
-
-# Ensure tmp folder always exists.
-mkdir -p /app/web/sites/default/files/private/tmp/
+# The location of the application directory.
+APP="${APP:-$APP}"
```

Comment:
this allows to use this script relative to any dir specified by `APP`. also, without it, no unit/integration testing is possible.

Reply:
Where is $APP being set on Lagoon (AFAIK it currently isn't) - should this not default to /app?

Reply:
good catch - it is supposed to be `/app`. amending